Add info on custom port for private registry image

sheepcount opened the pull request:
I still think it is a bug in the portal, but according to the Uservoice posting, it is not (https://feedback.azure.com/forums/169385-web-apps/suggestions/31304650). So the docs should at least give a hint on how to work around this issue.

azurecla commented:
Hi __@ShawnJackson__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (v-shawja). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

Commit:
Add info on custom port for private registry image

I still think it is a bug in the portal, but according to the Uservoice posting, it is not (https://feedback.azure.com/forums/169385-web-apps/suggestions/31304650). So the docs should at least give a hint on how to work around this issue.

ahmedelnably reviewed (approved):
Thanks for your valuable contribution
